src/librbd/operation/SnapshotProtectRequest.cc

```diff
@@ -33,7 +33,7 @@ SnapshotProtectRequest<I>::SnapshotProtectRequest(I &image_ctx,
                                                   Context *on_finish,
 						  const cls::rbd::SnapshotNamespace &snap_namespace,
 						  const std::string &snap_name)
-  : Request<I>(image_ctx, on_finish), m_snap_namespace(snap_namespace), m_snap_name(snap_name) {
+  : Request<I>(image_ctx, on_finish), m_snap_namespace(snap_namespace), m_snap_name(snap_name), m_state(STATE_PROTECT_SNAP) {
```

Comment:
Nit: wrap it down to the next line (with proper indentation) before it exceeds 80 characters